index.html

```diff
@@ -0,0 +1,50 @@
+<!DOCTYPE html>
+<!--[if lt IE 7]>      <html class="no-js lt-ie9 lt-ie8 lt-ie7"> <![endif]-->
+<!--[if IE 7]>         <html class="no-js lt-ie9 lt-ie8"> <![endif]-->
+<!--[if IE 8]>         <html class="no-js lt-ie9"> <![endif]-->
+<!--[if gt IE 8]>      <html class="no-js"> <!--<![endif]-->
+<html>
+  <head>
+    <meta charset="utf-8" />
+    <meta http-equiv="X-UA-Compatible" content="IE=edge" />
+    <title></title>
+    <meta name="description" content="" />
+    <meta name="viewport" content="width=device-width, initial-scale=1" />
+    <link rel="stylesheet" href="style.css" />
+  </head>
+
+  <body id="main">
+    <h1 id="title">Ada Lovelace mother of computer programming</h1>
+    <div id="img-div">
+      <img id="image" src="Assets/ada-lovelace.jpeg" width="800" height="400" />
+      <figcaption id="img-caption">Portrait of Ada Lovelace</figcaption>
+    </div>
+    <div id="story-p">
+      <p id="para">
+        Lived 1815 – 1852. Born two centuries ago, Ada Lovelace was a pioneer of
+        computing science. She took part in writing the first published program
+        and was a computing visionary, recognizing for the first time that
+        computers could do much more than just calculations. Lovelace was a
+        visionary: she understood that numbers could be used to represent more
+        than just quantities, and a machine that could manipulate numbers could
+        be made to manipulate any data represented by numbers. Lovelace died of
+        cancer in 1852, when she was only 36. More than 150 years later, we
+        remember her contributions to science and engineering in the celebration
+        of Ada Lovelace Day on October 13. First celebrated in 2009 (in March),
+        it is a day set aside to learn about women in science, technology,
+        engineering, and mathematics.
+      </p>
+    </div>
+    <p id="tribute-info">
+      Tribute page about Ada Lovelace, pioneer computer scientist
+    </p>
```

Comment:
This is a good use of semantic HTML, and it'd be even better if you wrap the text inside a div tag.